warehouse/seeds/_seeds.yml

```diff
@@ -81,3 +81,16 @@ seeds:
       - name: severity
         tests:
           - not_null
+
+  - name: ntd_agency_to_organization
+    description: Extract of Airtable mapping from NTD agency info to transit database organizations.
```

Comment:
Can you add a comment about versioning / how & when this matching was done? And why are you using organization name rather than `source_record_id`?

Reply:
I wanted this to be more human-maintainable and I didn't expect raw Airtable record IDs to be particularly user-friendly; also name has more semantic meaning here. I think my main question is whether you would prefer that the record ID or the org name be the join identifier?

Reply:
Joining on name-y strings always makes me a bit squirrelly about things like punctuation and capitalization (especially because organizations have a name within the NTD source that is not guaranteed to the be the same as our organization name). So I feel like record ID is safer, but I understand it's not very user-friendly. Sorry that's not conclusive.

Reply:
I'll keep both in the seed but join on record_id for now.